docker-entrypoint.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+set -e
+
+if [ -f /app/tmp/pids/server.pid ]; then
+  rm /app/tmp/pids/server.pid
+fi
+
+# this would blow up the db if the migration failed for any reason.
+# prob not the best approach!
+# bundle exec rake db:migrate 2>/dev/null || bundle exec rake db:setup
```

Comment:
seems like there's some debate about this.

Docker creates the DB automatically via the Postgres image when POSTGRES_DB is in the env.

But we still need to migrate and run a bootstrap task on first deploy, and then we need to run migrations whenever there are new ones. Can we use the image version to accomplish this?

I'd prefer that it be (safely) automated - we don't want to have to instruct users to run rake tasks.